todo-app/src/Components/Container.js

```diff
@@ -0,0 +1,17 @@
+import React, {Component} from 'react';
+
+export default class Container extends Component {

```

Comment:
I don't think `Container` is the most accurate name we can come up with here. What does this component do? It renders a `<ul>` of todo `<li>`s. How can we rename this to be more accurate?